CHANGELOG.md

```diff
@@ -1,10 +1,15 @@
+## Unreleased
+
+### Added
```

Comment:
I think this should be under `Fixed` section, state like `fix xxx issue/bug`, we do not need to expose how we fix it.

Reply:
changed